WordPress/WordPressTest/SiteCreation/SiteCreationRotatingMessageViewTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+@testable import WordPress
+
+class SiteCreationRotatingMessageViewTests: XCTestCase {
+    private let testMessages = [
+        "Test Message 1",
+        "Test Message 2",
+        "Test Message 3"
+    ]
+
+    private var rotatingMessageView: SiteCreationRotatingMessageView?
+
+    override func setUp() {
+        super.setUp()
+
+        self.rotatingMessageView = SiteCreationRotatingMessageView(messages: testMessages,
+                                                                   iconImage: UIImage())
+    }
+
+    override func tearDown() {
+        self.rotatingMessageView = nil
+
+        super.tearDown()
+    }
+
+    /// Make sure the view was properly configured with the provided status messages
+    func testSiteCreationRotatingMessageView_Initialize() {
+        XCTAssertEqual(rotatingMessageView?.statusMessages, testMessages)
```

Comment:
Maybe we can get rid of this test?

The test below checks if the messages are appearing on the `statusLabel` — which is what really matters to us. The `statusMessages` is just an implementation detail.

That makes sense?

Reply:
Makes sense to me! My thinking was checking to see if the class was initialized correctly with the given status messages.